cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.adoc

```diff
@@ -4,37 +4,28 @@
 //Review any deprecation notices and policy changes planned in the next Prisma Cloud Code Security release.
 
 Read this section to learn about what is planned in the upcoming release. The Look Ahead announcements are for an upcoming or next release and it is not a cumulative list of all announcements.
-Currently, there are no previews or announcements for updates.
+//Currently, there are no previews or announcements for updates.
 
-//NOTE: The details and functionality listed below are a preview and the actual release date is subject to change.
+NOTE: The details and functionality listed below are a preview and the actual release date is subject to change.
```

Comment:
The details and functionality listed below are a preview. The actual release date is subject to change.